### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by Mapsred**
@TavernOfCho/cho-core-team This PR is opened to discussion

### CODE REVIEW

**Review by Qtipy — APPROVED** [review]

File: src/translation/fr.json [hunk 1]
```diff
@@ -26,34 +26,34 @@
...
+    "helper.title": "Gardez votre calme et suivez mes instructions.",
     "homescreen": "Taverne",
-    "homescreen.welcome": "Admirons ensemble votre collection, planifier vos prochains exploits et trouver d'autres aventuriers",
+    "homescreen.welcome": "Admirons ensemble votre collection, planifiez vos prochains exploits et trouvez d'autres aventuriers.",
     "herounit.achievement": "Choisir un nouveau haut-fait à réaliser",
     "herounit.battlepet": "Découvrir une nouvelle mascotte",
     "herounit.character": "Rencontrer de nouvelles personnes",
     "herounit.description": "Bienvenue dans la taverne de Cho !",
     "herounit.explore": "Et si on discutait ! Que veux-tu savoir aujourd'hui ?",
-    "herounit.help": "Je suis perdu(e) ! :(",
```
Comment (AxelE) [inline src/translation/fr.json:48]: @Mapsred :girl:

File: src/translation/fr.json [hunk 2]
```diff
@@ -26,34 +26,34 @@
...
     "herounit.mount": "Découvrir une nouvelle monture",
-    "herounit.pleaseregister": "Je me présente, je suis le chroniqueur Cho. Quel est votre nom déjà ?",
-    "issue": "Rapporter une erreur",
+    "herounit.pleaseregister": "Je me présente, je suis le chroniqueur Cho. Comment vous appelez-vous ?",
+    "issue": "Signaler une erreur",
     "tavernproject": "Projet TavernOfCho",
     "title.achievement": "Voulez-vous que je vous conte une légende ?",
-    "title.battlepet": "Je connais toutes les sortes de mascottes, essayez pour voir !",
-    "title.character": "Je connais tout le monde, envi(e) de faire un essai ?",
-    "title.help": "Perdu(e) ? Ne vous inquietez pas !",
```
Comment (AxelE) [inline src/translation/fr.json:56]: @Mapsred :woman:

**Review by AxelE — CHANGES_REQUESTED** [review]
:woman_cartwheeling:

File: src/translation/fr.json (hunk 1 above)
Comment (Mapsred) [reply src/translation/fr.json:48]: :wave:

**Review by Qtipy — APPROVED** [review]
A good french guy

**Comment by AxelE** [comment]
:nauseated_face:

**Pull request merged**